pkg/strongswan/Dockerfile

```diff
@@ -0,0 +1,79 @@
+#
+# StrongSwan VPN + Alpine Linux
+#
+
+FROM alpine:3.6 as build
+
+ENV STRONGSWAN_RELEASE https://download.strongswan.org/strongswan.tar.bz2
+
+RUN apk add --no-cache \
+    gcc \
+    make \
+    patch \
+    libc-dev \
+    linux-headers \
+    tar
+
+RUN mkdir -p /tmp/strongswan
+RUN mkdir -p /out
+
+WORKDIR /tmp/strongswan
+
+RUN apk --update add build-base \
+            ca-certificates \
+            iptables \
+            iproute2 \
+			openssl \
+			openssl-dev \
+            curl\
+            curl-dev
+
+RUN  curl -Lo /tmp/strongswan/strongswan.tar.bz2 $STRONGSWAN_RELEASE 
+RUN  tar --strip-components=1 -C /tmp/strongswan -xjf /tmp/strongswan/strongswan.tar.bz2
+RUN  cd /tmp/strongswan
```

Comment:
Why is this cd here if you have WORKDIR /tmp/strongswan above?

Reply:
Sure, will change